src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -167,11 +160,16 @@ public function setupFieldAssociations($fields, $table, $fieldConditions, $strat
             ]);
         }
 
+        $conditions = ["$targetAlias.model" => $model];
+        if (!$this->_config['allowEmptyTranslations']) {
+            $conditions["$targetAlias.content !="] = '';
+        }
```

Comment:
This needs to be 
```
if (!$this->_config['allowEmptyTranslations']) {
	$conditions["$targetAlias.content <>"] = '';
}
```
Since that'll check for empty or null.